docs/explanation/reader-explanation.md

```diff
@@ -0,0 +1,274 @@
+# STS Reader
+The prime purpose of the reader is to transform data from measurement files into community-defined concepts constructed by the SPM community which allows experimentalists to store, organize, search, analyze, and share experimental data (only within the [NOMAD](https://nomad-lab.eu/nomad-lab/) platform) among the scientific communities. 
+
+## Supproted Files and File Versions
+- Reader builds on [NXsts](https://github.com/FAIRmat-NFDI/nexus_definitions/blob/fairmat/contributed_definitions/NXsts.nxdl.xml) application definition
+- Needs an experimental file, a config file and a eln file
+- Can parse Scanning Tunneling Spectroscopy (STS) from
+    - Nanonis: Generic 5e, Generic 4.5
+- Can parse Scanning Tunneling Microscopy (STM) from
+    - Nanonis: Generic 5e, Generic 4.5
+
+## NeXus Application Definition
+To define a standarized schema, we choose NeXus ontology and we defined a application definition `NXsts` for reading and standarizing `STM` as well as `STS` experimental data. To get the application definition and other related base classes please follow NeXus-FAIRmat page for [Scanning Tunneling Spectroscopy](https://fairmat-nfdi.github.io/nexus_definitions/sts-structure.html). There you will get application definition `NXsts` and other base classed used to construct the application defintion.
+
+## Introduction to the Different Input Files
+To utilize or reuse or extend the reader, motivation behind the reader input files must be understood. The files are using specific semantics rules so that reader can understand the files and work with the content of files.
```

Comment:
```suggestion
To utilize, reuse, or extend the reader, the different reader input files must be understood. The files are using specific semantic rules so that reader can understand the files and work with their content.
```

Reply:
Resolved!